uber/static/deps/combined.js

```diff
@@ -103214,3 +103214,114 @@ S2.define('jquery.select2',[
         window['toastr'] = factory(window['jQuery']);
     }
 }));
+
+/* ========================================================================
```

Comment:
Huh, that's weird. We should have had this stuff already. We should look into whether this is duplicate code.